js/utils/helpers.js

```diff
@@ -1850,11 +1850,14 @@
                         if (attrs[0].children[0].type == "link_open") {
                             var iframeHTML = "<iframe ", openingLink = attrs[0].children[0];
                             var enlargeLink, posTop = true, captionClass = "", captionStyle = "", iframeClass = "", iframeStyle = "";
+                            var isYTlink = false, newClass='class="' + "hide-in-print" + '"', videoURL = "";
```

Comment:
The last time that I looked at your code I forgot to mention this.

 In ermrestjs, we define all the constants in `js/utils/constant.js` file.  If you search for `module._classNames` we actually have a list of classes as well. These two classes that you're adding should be defined there. 

Notice how in the `video` tag implementation we're using the `module._classNames.postLoad`.